elements/lisk-dpos/src/dpos.ts

```diff
@@ -41,13 +42,15 @@ import {
 	ForgersList,
 	StateStore,
 	DecodedVoteWeights,
-	DecodedUsernames,
 } from './types';
 
 interface DposConstructor {
 	readonly chain: Chain;
+	readonly initDelegates: ReadonlyArray<Buffer>;
 	readonly activeDelegates?: number;
 	readonly standbyDelegates?: number;
+	readonly genesisHeight?: number;
```

Comment:
```suggestion
	readonly genesisBlockHeight: number;
```

It should not be optional for DPOS, must be provided from the genesis block. Its already optional `createGenesisBlock`, so should not be redundant logic here.